tests/test_dataloaders.py

```diff
@@ -0,0 +1,87 @@
+from datasets import Dataset, load_dataset
+
+from evaluate import evaluator
+
+
+def test_evaluator_text_classification():
+    e = evaluator("text-classification")
+
+    # Test passing in dataset object
+    data = Dataset.from_dict(load_dataset("imdb")["test"][:2])
```

Comment:
simpler this way ;)
```suggestion
    data = load_dataset("imdb", split="test[:2]")
```
unless you wanted a raw Dataset object ? (no metadata in dataset.info for example)